src/hooks/hooks.ts

```diff
@@ -18,28 +19,27 @@ AfterAll(async () => {
   await browser.close();
 });
 
-Before(async function (this: any) { // Access 'this' for scenario context
-  console.log('Creating new context and page...');
-  Context = await browser.newContext();
-  const page = await Context.newPage();
-  pageFixture.page = page; 
+Before(async function (this: any) {
+  // Create context with stored credentials for admin
+  adminContext = await browser.newContext({
+    storageState: authConfig.admin.storageState
+  });
+  adminPage = await adminContext.newPage();
+  pageFixture.adminPage = adminPage;
 
-  // Store the page in the Cucumber World for access in steps
-  this.page = page; 
+  console.log("Creating new contexts and pages with auth...");
+  this.page = adminPage; // Default to admin page
 });
 
 After(async function (this: any, { pickle, result }) {
   console.log("Closing context and page...");
   if (result?.status === Status.FAILED) {
+    console.log("Taking screenshot...");
```

Comment:
Please remove all the console.log statements, as they will log to the browser console instead of the terminal.

Reply:
Done